src/Symfony/Component/Serializer/Tests/Normalizer/AbstractObjectNormalizerTest.php

```diff
@@ -416,6 +493,36 @@ public function instantiateObject(array &$data, string $class, array &$context,
     }
 }
 
+class ObjectWithStringBoolean
+{
+    /** @var bool */
+    public $true;
```

Comment:
```suggestion
    public bool $true;
```

Unthinkable we can use this, cc @derrabus @nicolas-grekas

Reply:
I decided not to use type as not a single class in there had types

Reply:
I agree, but the reason is, they came from Symfony versions which does not support php versions with types.

@derrabus @nicolas-grekas please decide, thanks

Reply:
Yes, I do agree with that, but in my opinion, having one class with types in the whole file doesn't really make sense and it's best to either update the whole file or don't change.

P.S. I don't know if you noticed but the test case uses  PhpDocExtractor so property types would not do anything, and would require implementing a different extractor.

Reply:
@OskarStark We still support PHP 7.2 on the 5.4 branch, so no typed properties please.

Reply:
Thanks 👍🏻